docs/arvo/network.md

```diff
@@ -88,7 +86,7 @@ line).
 The second thing `++poke-noun` produces is our state.  `+>.$`
 refers to a particular address in our subject where our formal
 app state is stored.  It'll become clear why this is later on,
-but for now pretend that `+>.$` is a magic invocation that means
+but for now pretend that `+>.$` is a magic incantation that means
 "app state".
```

Comment:
Oh now that I see the context maybe you're right.  You're the original author — this is really up to you!